### ISSUE TITLE
[css-ui] Add caret-width or add spec that browser maintains caret width despite transforms

### ISSUE THREAD

**Issue by shalanah** [w3c/csswg-drafts#4784]
https://www.w3.org/TR/css-ui-4/#insertion-caret

Adding `caret-width` or adding to the spec that browsers should maintain caret width despite transforms of editable text would be awesome. 

In my particular case, carets are not visible to the user since I have transformed and scaled-down my text. Although the text is still very visible, the caret is not because it is now in sub-pixel territory.

**Comment by frivoal** [comment]
I wonder if there's a real need to control the size of the caret explicitely other than to keep it visible. If that's just the latter, maybe the fact that it becomes invisible in the situation you described can simply be considered a bug, and browsers should fix it, with no need for a new property and for authors to do anything explicitely.

Do you have other scenarios where you feel the ability to control the size of the caret would be useful?

**Comment by shalanah** [comment]
This is an all-browser bug then! On scaling up the cursor also gets a larger width across browsers. I don't think they are going to change that behavior without prompting, but adding a `caret-width` might help. 

The more I think about it... the more I want from my caret!

In order to remedy the issue of disappearing caret, I had to use a `div` and place it correctly. I had so much more freedom styling a div than I did with a caret. For instance, I can use `backdrop-filter` so the caret always contrasts with the background image or color.

I think adding `caret-shape` shows the want to edit this elem more and `caret-shape` and even `caret-width` might be bandaids to surfacing more design freedom.
